go.mod

```diff
@@ -11,32 +11,33 @@ require (
 	github.com/hashicorp/vault v1.7.0
 	github.com/hashicorp/vault-plugin-auth-gcp v0.9.0
 	github.com/hashicorp/vault/api v1.1.0
-	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/sapmexporter v0.31.0
-	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/signalfxexporter v0.31.0
-	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/splunkhecexporter v0.31.0
-	github.com/open-telemetry/opentelemetry-collector-contrib/extension/httpforwarder v0.31.0
-	github.com/open-telemetry/opentelemetry-collector-contrib/extension/observer/hostobserver v0.31.0
-	github.com/open-telemetry/opentelemetry-collector-contrib/extension/observer/k8sobserver v0.31.0
-	github.com/open-telemetry/opentelemetry-collector-contrib/extension/storage v0.31.0
-	github.com/open-telemetry/opentelemetry-collector-contrib/pkg/experimentalmetricmetadata v0.31.0
-	github.com/open-telemetry/opentelemetry-collector-contrib/processor/k8sprocessor v0.31.0
-	github.com/open-telemetry/opentelemetry-collector-contrib/processor/metricstransformprocessor v0.31.0
-	github.com/open-telemetry/opentelemetry-collector-contrib/processor/resourcedetectionprocessor v0.31.0
-	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/carbonreceiver v0.31.0
-	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/collectdreceiver v0.31.0
-	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/filelogreceiver v0.31.0
-	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/fluentforwardreceiver v0.31.0
-	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/k8sclusterreceiver v0.31.0
-	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/kafkametricsreceiver v0.31.0
-	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/kubeletstatsreceiver v0.31.0
-	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/prometheusexecreceiver v0.31.0
-	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/receivercreator v0.31.0
-	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/redisreceiver v0.31.0 // indirect
-	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/sapmreceiver v0.31.0
-	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/signalfxreceiver v0.31.0
-	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/simpleprometheusreceiver v0.31.0
-	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/splunkhecreceiver v0.31.0
-	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/statsdreceiver v0.31.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/sapmexporter v0.30.1-0.20210727182931-b5e442c20695
```

Comment:
falling back to a v0.30.1 pseudo version isn't desirable.

Reply:
I will change to v0.31.0